apps/prairielearn/public/stylesheets/local.css

```
@@ -154,6 +154,26 @@ mjx-container {
   --bs-btn-disabled-border-color: #f8f9fa;
 }
 
+/**
+ * Similar to `btn-light`, but with a slightly darker background.
```

Nit, some breadcrumbs for our future selves (this will probably matter once we start pursuing dark mode support):

```suggestion
 * A darker variant of `btn-light`.
 *
 * These colors were determined by rebuilding the Bootstrap CSS with the `light`
 * base color redefined from gray-100 to gray-300 here:
 *
 * https://github.com/twbs/bootstrap/blob/bafa9d25ed38b8742a3f52bc1401d46b71960153/scss/_variables.scss#L307
```